src/contracts/EthenaARM.sol

```diff
@@ -17,9 +17,17 @@ contract EthenaARM is Initializable, AbstractARM {
     IERC20 public immutable usde;
     /// @notice The address of Ethena's staked synthetic dollar token (sUSDe)
     IStakedUSDe public immutable susde;
+    /// @notice The maximum number of unstaker contracts that can be used per day of the week
+    uint8 public constant MAX_REQUESTS_PER_DAY = 10;
```

Comment:
I don't think we need to add the extra complexity of restricted a max per day. If there a 70 and some days use more than 10 another others less, then that's fine